src/components/plots/xyPlot.ts

```diff
@@ -71,6 +71,49 @@ module Plottable {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      if (attr === "x") {
+        return super._updateExtentsForAttr(attr, this._xFilter());
+      } else if (attr === "y") {
+        return super._updateExtentsForAttr(attr, this._yFilter());
+      }
+      return super._updateExtentsForAttr(attr);
+    }
```

Comment:
Hmm... I'm getting a really a nervous feeling about this design...  Can't put my finger on it, but this seems really weird...